### DISCUSSION TITLE
rainbowkit v1.3.x doc wanted for adding custom wallet app on the wallet list.

### DISCUSSION THREAD

**Discussion by bsistuk** [rainbow-me/rainbowkit#1857] [Q&A] [answered]
Hi,

My repository was using rainbowkit v1.3.0. I need to add our custom wallet but following v2 doc doesn't work out.
Could you please share a solution of this on v1.3 or a legacy documentation?

Thank you.

**Comment by 0xmkh** [comment, marked as answer]
@bsistuk Yes we have a legacy documentation for `1.3.x` versions. Check it out here: https://v1.rainbowkit.com/docs/custom-wallets

**Reply by bsistuk** [reply]
Thank you so much!
